flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -362,14 +368,17 @@ class FMCMainDisplay extends BaseAirliners {
                     CDUProgressPage.ShowPage(this);
                 }
 
-                let preSelectedClbSpeed = this.preSelectedClbSpeed;
-
-                if (SimVar.GetSimVarValue("L:A32NX_GOAROUND_PASSED", "bool") === 1) {
-                    preSelectedClbSpeed = this.computedVgd;
+                /** Activate pre selected speed/mach */
+                // this.activatePreSelSpeedMach(_lastFlightPhase === FmgcFlightPhases.GOAROUND ? ... : this.preSelectedClbSpeed);
```

Comment:
Is this comment still needed? Looks like old logic you replaced.

Reply:
Yes the comment is about the old logic, it would've preselected GD speed during GO AROUND.

I haven't yet checked this against the FCOM/manual.